src/librustc_trans/save/dump_csv.rs

```diff
@@ -1118,10 +1178,11 @@ impl<'l, 'tcx, 'v> Visitor<'v> for DumpCsvVisitor<'l, 'tcx> {
     }
 
     fn visit_mac(&mut self, _: &ast::Mac) {
-        // Just stop, macros are poison to us.
+        // These shouldn't exist in the AST at this point, do nothing.
```

Comment:
Should we warn or assert here? Seems like we might want to at least have some debug! logging